scripts/local_testnet/beacon_node.sh

```diff
@@ -67,5 +67,4 @@ exec $lighthouse_binary \
 	--target-peers $((BN_COUNT - 1)) \
   --execution-endpoint $execution_endpoint \
   --execution-jwt $execution_jwt \
```

Comment:
Do we want to remove the backlash in the previous line?
```suggestion
  --execution-jwt $execution_jwt 
```

Reply:
thanks, removed in [a3e4f5e](https://github.com/sigp/lighthouse/pull/5500/commits/a3e4f5e57661640efc778b37dca509c8484700b5)

Reply:
no, we need to keep it because `$BN_ARGS` is on the next line